CHANGELOG.next.asciidoc

```diff
@@ -212,6 +212,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update Logstash module's Grok patterns to support Logstash 7.4 logs. {pull}14743[14743]
 - Fix a problem in Filebeat input httpjson where interval is not used as time.Duration. {issue}14752[14752] {pull}14753[14753]
 - Fix SSL config in input.yml for Filebeat httpjson input in the MISP module. {pull}14767[14767]
+- Fix a breaking change in the Zeek module connection fileset where the field mapping of "connection.state" was changed. {pull}15169[15169]
```

Comment:
Given that this change that you're fixing was never released I think there shouldn't be a changelog entry. It's probably confusing to tell the end-user we fixed a breaking change that they were never affected by.

Reply:
SGTM